data/ideas.json

```diff
@@ -214,6 +214,12 @@
         "description": "App for share knowledge by donationg books.",
         "level": "intermediate",
         "tags": "book-donation"
+    },
+    {
+        "title": "Guess the Number",
+        "description": "Guess the number between 1 - 20",
+        "level": "basic",
+        "tags": "number-guessing, JavaScript, html, css"
```

Comment:
Please remove the programming language due to the idea could make in another language.

Reply:
I have removed the languages from ideas